### ISSUE TITLE
Iterate over the priorities and elements in a priority queue directly

### ISSUE THREAD

**Issue by Xnartharax** [JuliaCollections/DataStructures.jl#749]
I think a convenience function to iterate only over the priorities or the elements of the queue similar to `keys` and `values` for dicts would be nice to have. The naming could be something like `priorities` and `values`. I'd also be willing to implement this feature. This would be my first contribution to an open-source project.

**Comment by oxinabox** [comment]
If I recall correctly `keys` and `values` does that already, no?

**Comment by Xnartharax** [comment]
Yes you're right. I didn't check that. However, it is not documented.

**Comment by oxinabox** [comment]
A PR to document that would be appreciated.
